### PR TITLE
Export Uniform Hazard Spectra

### PR DESCRIPTION
**Pull request by micheles**
The XML exporter was missing in oq-lite and I have changed heavily the CSV exporter. Closes https://github.com/gem/oq-risklib/issues/557

### CODE REVIEW

File: openquake/calculators/calc.py [hunk 1]
```diff
@@ -221,7 +221,20 @@ def gmvs_to_haz_curve(gmvs, imls, invest_time, duration):
...
 # ################## utilities for classical calculators ################ #
 
-def make_uhs(maps):
+def get_imts_periods(imtls):
+    """
+    Returns a list of IMT strings and a list of periods. There is an element
+    for each IMT of type Spectral Acceleration (PGA is considered an alias
+    for SA(0.0))
+
+    :param imtls: as set of intensity measure type strings
```
Comment (ptormene) [inline openquake/calculators/calc.py]: typo as -> a (I suppose)

File: openquake/calculators/calc.py [hunk 2]
```diff
@@ -221,7 +221,20 @@ def gmvs_to_haz_curve(gmvs, imls, invest_time, duration):
...
+    """
+    imts = sorted(from_string(imt) for imt in imtls
+                  if imt.startswith('SA') or imt == 'PGA')
+    return map(str, imts), [imt[1] or 0.0 for imt in imts]
```
Comment (ptormene) [inline openquake/calculators/calc.py]: It looks like this is returning something different with respect to what described in the docstring

File: openquake/calculators/calc.py (hunk 1 above)
Comment (micheles) [inline openquake/calculators/calc.py]: Yes

Comment (ptormene) [inline openquake/calculators/calc.py]: Please add the `:returns:` description

**Comment by ptormene** [comment]
LGTM

**Pull request merged**